doc/prometheus.md

```diff
@@ -33,8 +33,9 @@ rich telemetry data to your cluster.  Simply add the following item to your
       - source_labels:
         - __meta_kubernetes_pod_container_name
         - __meta_kubernetes_pod_container_port_name
+        - __meta_kubernetes_pod_label_linkerd_io_control_plane_ns
         action: keep
-        regex: ^linkerd-proxy;linkerd-metrics$
+        regex: ^linkerd-proxy;linkerd-metrics;linkerd$
```

Comment:
i think this `linkerd` is inconsistent with the `{{.Namespace}}` reference 20 lines above. using `linkerd` is nice because it's the default and it'll just work, using `{{.Namespace}}` is more correct but requires users to modify the yaml, even in the default case. either way, we should be consistent in both places.

Reply:
Ahhhh.... thanks -- I struggled with how to handle this and ultimately made the wrong choice. Will submit a fix.